man/reopen_connection.Rd

```diff
@@ -0,0 +1,26 @@
+\name{reopen_connection}
+\alias{reopen_connection}
```

Comment:
Add `\alias{}` for the implemented methods?

Reply:
done!